src/Propulsion.DynamoStore/DynamoStoreSource.fs

```diff
@@ -115,14 +115,27 @@ module private Impl =
 
 [<NoComparison; NoEquality>]
 type LoadMode =
+    /// Skip loading of Datta for events; this is the most efficient mode as it means the Source only needs to read from the index
```

Comment:
```suggestion
    /// Skip loading of Data for events; this is the most efficient mode as it means the <c>Source</c> only needs to read from the index
```

Reply:
FYI sadly the text needs to be wrapped in summary tags for the c tags to 'work' - I tend to use backticks instead (and should have done for Data instead